hangman.py

```diff
@@ -0,0 +1,270 @@
+            compared_list.append(i)
+        elif i not in letters_guessed:
+            compared_list.append("_ ")
+    compared_word = ""
+    return compared_word.join(compared_list)
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: set (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    alphabet_list = list(string.ascii_lowercase)
+    compared_list = [i for i in alphabet_list if i not in letters_guessed]
+    compared_alphabet = ""
+    return compared_alphabet.join(compared_list)
+
+
+def count(warn, tries):
+    '''
+    warnings: amount of warnings
+    tries: amount of tries
+    returns: list with amount of warnings and tries or messages, if there isn't any tries or warnings.
+    '''
+    count_list = []
+    if warn != 0 and warn != "no":
+        warn -= 1
+        count_list = [warn, tries]
+    else:
+        warn = "no"
+        tries -= 1
+        count_list = [warn, tries]
+    return count_list
+
+
+def vowel_check(letter, tries):
+    if letter in {"a", "e", "i", "o", "u"}:
+        tries -= 2
+    else:
+        tries -= 1
+    return tries
+
+
+def check_element(tries, warn, letters_guessed, check, letter, hints):
+    # This loop checks if the entered symbol has already been used
+    if letter in letters_guessed:
+        warn, tries = count(warn, tries)[0], count(warn, tries)[1]
+        # This output changes depending on if the entered symbol is "*" or a letter and how many warnings are left
+        print("Oops! You've already", *["used your hint." if letter == "*" else "guessed that letter."], "You have", warn, "warnings left" + str(*[" so you lose one guess" if warn == "no" else ""]) + ":", check)
+
+    # If the entered symbol is a "*", the program uses a hint and shows all possible matches
+    elif hints and letter == "*":
+        show_possible_matches(check)
+
+    # Then it checks if the entered symbol is a letter
+    elif letter not in string.ascii_lowercase:
+        warn, tries = count(warn, tries)[0], count(warn, tries)[1]
+        print("Oops! That is not a valid letter. You have", warn, "warnings left:", check)
+
+    # If the entered symbol is a letter and it's in the secret word, the program shows that the letter is guessed
+    elif letter in set(secret_word):
+        letters_guessed.add(letter)
+        check = get_guessed_word(secret_word, letters_guessed)
+        print("Good guess:", check)
+
+    # If the entered symbol is a letter and it isn't in the secret word, the program shows that the letter
+    # isn't guessed and user loses one try if it isn't a vowel letter or two if it is.
+    elif letter not in set(secret_word):
+        tries = vowel_check(letter, tries)
+        print("Oops! That letter is not in my word:", check)
+    letters_guessed.add(letter)
+    l = [tries, warn, secret_word, letters_guessed, letter, check]
```

Comment:
Are you sure that you need to return all of these values?
And I also will recommend you to return a tuple

Reply:
But you need to specify in the docstring what exactly your function returns